configure.go

```diff
@@ -0,0 +1,45 @@
+package discovery
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	kv "gopkg.in/clever/kayvee-go.v2"
+)
+
+// m is a convenience type for using kv.
+type m map[string]interface{}
+
+const (
+	templateVar = "SERVICE_%s_%s_%%s"
+)
+
+func getVar(envVar string) string {
+	envVar = strings.ToUpper(envVar)
+	val := os.Getenv(envVar)
+	if val == "" {
+		panic(kv.FormatLog("discovery-go", kv.Error, "missing env var", m{
```

Comment:
Environment variables should always be present when using this library. Do you think it is better to return an error?